core/src/main/java/org/mskcc/cbio/oncokb/model/InferredMutation.java

```diff
@@ -10,7 +10,8 @@
     ONCOGENIC_MUTATIONS("Oncogenic Mutations"),
     GAIN_OF_FUNCTION_MUTATIONS("Gain-of-function Mutations"),
     LOSS_OF_FUNCTION_MUTATIONS("Loss-of-function Mutations"),
-    SWITCH_OF_FUNCTION_MUTATIONS("Switch-of-function Mutations");
+    SWITCH_OF_FUNCTION_MUTATIONS("Switch-of-function Mutations"),
+    VUS_MUTATIONS("VUS Mutations");
```

Comment:
Let's do VUS("Variants of Unknown Significance")

Reply:
We also needs to support the abbreviation.
Check the file `abbreviation-ontology.tsv`